src/components/projectRegister/RegisterHeader.tsx

```
@@ -1,67 +1,311 @@
+  const strokeWidth = 4
+  const normalizedRadius = radius - strokeWidth * 2
+  const circumference = normalizedRadius * 2 * Math.PI
+  const strokeDasharray = `${circumference} ${circumference}`
+  const strokeDashoffset = circumference - (progress / 100) * circumference
+  // 필수 항목만 완료되면 체크마크 표시
+  const isCompleted = requiredCompleted === requiredTotal && requiredTotal > 0
+
+  return (
+    <div className={`flex items-center cursor-pointer ${isActive ? "opacity-100" : "opacity-50"}`} onClick={onClick}>
+      {/* 원형 게이지 - 크기 줄임 (w-24 h-24 -> w-20 h-20) */}
+      <div className="relative w-20 h-20 flex-shrink-0">
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)